problems/week3_SORT/연서/K번째수/solution.js

```diff
@@ -0,0 +1,30 @@
+function solution(array, commands) {
+  var answer = [];
+
+  // commands 중첩 배열 분리를 위한 반복문
+  for (let i = 0; i < commands.length; i++) {
+    // first : 자를 시작 인덱스
+    // (인덱스를 1,2,3,.. 이렇게 1부터 셌으므로 slice에 넣을 때 +1로 생각)
+    let first = commands[i][0];
+    // last  : 자를 마지막 인덱스
+    // (마찬가지로 인덱스를 1부터 셌으므로 slice에 넣을 때 +1을 안해줘도 last번째 요소까지 포함)
+    let last = commands[i][1];
+    // 정렬 후 찾을 index (target index)
+    // (마찬가지로 인덱스를 1부터 셌으므로 index를 찾을 때 -1을 해야 함)
+    let t = commands[i][2];
```

Comment:
```suggestion
    let [first, last, t] = commands[i];
    // 위 방식대로 forEach로 대체한다면 commans[i] 말고 item
```
요렇게 구조분해할당 하는게 더 깔끔하겠죠잉?

Reply:
구조분해할당을 놓쳤당 다음에 비슷한 케이스가 있다면 꼭 반영할게용🖤

Reply:
위에서 묶어놓고 왔는데 아래에 이미 발빠른 소녀가 해뒀군....